app/modules/encounters/resources.py

```diff
@@ -265,59 +263,3 @@ def delete(self, encounter):
         # TODO handle failure of feather deletion (when edm successful!)  out-of-sync == bad
         encounter.delete()
         return None
-
-
-def _request_passthrough(target, path, request_func, passthrough_kwargs):
```

Comment:
definitely these `request_passthrough()` calls were chief redundant thing i was dropping all over.  thanks for catching and corralling these.